apps/api_web/lib/api_web/views/schedule_view.ex

```diff
@@ -61,10 +61,16 @@ defmodule ApiWeb.ScheduleView do
     State.Prediction.prediction_for(schedule, date)
   end
 
+  def arrival_time(%{drop_off_type: 1}, %{assigns: %{api_version: ver}}) when ver >= "2019-07-01",
```

Comment:
Per Paul's comment, I'd do this the other way:
1) Update the parsing to have the data in the format we want
2) In the view, if we have an older version and a missing time, then copy it from the `departure_time` (and vice versa)

Reply:
Done.